compiler/rustc_codegen_ssa/src/back/linker.rs

```diff
@@ -1116,10 +1122,6 @@ impl<'a> Linker for MsvcLinker<'a> {
             // straight to exports.
             writeln!(f, "LIBRARY")?;
             writeln!(f, "EXPORTS")?;
-            for symbol in symbols {
```

Comment:
I don't believe that we need the def file any more, we should be able to get away with passing `/DLL` to the linker.

Reply:
In the mingw docs I read something about the presence of a .DEF file disabling auto-export of certain symbols. I don't know if the same applies to MSVC, so I added it here just to be safe.

Reply:
Yes, this is an ld.bfd (and LLD in MinGW mode) feature to ease porting of Unix code, but it is not only .DEF that disables it. Any supported ld.bfd way to explicitly export symbols would suffice, and there is a CLI flag for that as well.

None of that matters here because link.exe doesn't export everything by default.